src/main/java/controller/Controller.java

```diff
@@ -0,0 +1,100 @@
+package controller;
+
+import domain.Reserve;
+import domain.Time;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@org.springframework.stereotype.Controller
```

Comment:
```suggestion
@Controller
```

깔끔하게 어노테이션 이름만 보이게 해보는건 어떨까요?